crates/fluvio-spu/src/core/global_context.rs

```diff
@@ -478,5 +481,41 @@ mod file_replica {
                 error!("leader switch failed: {}", err);
             }
         }
+
+        /// Delete consumers offset for given replica if it is leader consumer
+        async fn delete_consumers_offset(&self, replica: &Replica) -> anyhow::Result<()> {
+            let consumers_replica_id =
```

Comment:
this should be const since this is same for all invocation

Reply:
move to `default.rs` as tuple to not import fluvio-protocol there.

so we can now use it as `CONSUMER_REPLICA_KEY.into()`